colcon_cargo/task/cargo/test.py

```diff
@@ -22,7 +24,12 @@ def __init__(self):  # noqa: D107
         satisfies_version(TaskExtensionPoint.EXTENSION_POINT_VERSION, '^1.0')
 
     def add_arguments(self, *, parser):  # noqa: D102
-        pass
+        parser.add_argument(
+            '--cargo-args',
+            nargs='*', metavar='*', type=str.lstrip,
+            help='Pass arguments to Cargo projects. '
+            'Arguments matching other options must be prefixed by a space,\n'
+            'e.g. --cargo-args " --help"')
 
     async def test(self, *, additional_hooks=None):  # noqa: D102
```

Comment:
Can we actually add a docstring here? Perhaps just mentioning at a high level the two cargo commands running and why?

Reply:
I'm not sure what to say about the why we run tests / fmt :sweat_smile: but very valid, I added a docstring saying what we run (test and fmt) and what we don't and why we don't (docs) https://github.com/colcon/colcon-cargo/pull/39/commits/bf3637ef0f4f3259125c3b415452cf53c660b23b